src/components/TodoContainer.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import TodosList from './TodosList';
+import Header from './Header';
+import InputTodo from './InputTodo';
+
+class TodoContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      todos: [
+        {
+          id: 1,
+          title: 'Setup development environment',
+          completed: false,
+        },
+        {
+          id: 2,
+          title: 'Develop website and add content',
+          completed: false,
+        },
+        {
+          id: 3,
+          title: 'Deploy to live server',
+          completed: false,
+        },
+      ],
+    };
+  }
+
+  handleChange = (id) => {
+    this.setState((prevState) => ({
+      todos: prevState.todos.map((todo) => {
+        if (todo.id === id) {
+          return {
+            ...todo,
+            completed: !todo.completed,
+          };
+        }
+        return todo;
+      }),
+    }));
+  };
+
+  delTodo = (id) => {
+    this.setState((prevState) => ({
+      todos:
+        prevState.todos.filter((todo) => todo.id !== id),
+    }));
+  };
+
+  addTodoItem = () => {
+
+  };
+
+  render() {
+    const { todos } = this.state;
+    return (
+      <div>
```

Comment:
- You have done an awesome 💯 job using JSX and the `render` life-cycle function but since the `return` statement of the `render()` method can only return one  1️⃣  JSX element, you have used a `div` as a container for the whole App to solve this problem. But this has a side-effect 📛  of filling your app with redundant non-semantic code. Instead, you can wrap your components in a `<React.Fragment>` or the empty tags as containers (`<>`) & it is not going to show as real tags in the render time:

- using `<React.Fragment>`:

  ```suggestion
     <React.Fragment>
  ```

  ```js
  return(
    <React.Fragment>
      <Header />
      <InputTodo addTodoProps={this.addTodoItem} />
      <TodosList
           todos={todos}
           handleChangeProps={this.handleChange}
           deleteTodoProps={this.delTodo}
       />
    </React.Fragment>
  )
  ```

- using empty tags `<>`

  ```suggestion
      <>
  ```

  ```js
  return(
    <>
      <Header />
      <InputTodo addTodoProps={this.addTodoItem} />
      <TodosList
        todos={todos}
        handleChangeProps={this.handleChange}
        deleteTodoProps={this.delTodo}
      />
    </>
  )
  ```

Reply:
It is a great suggestion, thanks 💯 .